boxoffice/views/admin_discount.py

```diff
@@ -52,12 +62,38 @@ def jsonify_discount_policies(data_dict):
     )
 @render_with({'text/html': 'index.html', 'application/json': jsonify_discount_policies}, json=True)
 def admin_discount_policies(organization):
+    RESULTS_PER_PAGE = 6
     query = request.args.get('search')
-    if query:
-        discount_policies = DiscountPolicy.query.filter(DiscountPolicy.title.ilike('%{query}%'.format(query=query))).all()
+    try:
+        page = int(request.args.get('page', 1))
+    except ValueError:
```

Comment:
@shreyas-satish ah. ok. I'm handling both the case of `page` being present and the chance that it might not be a number. In the simple if/else, it'll throw exception if page is not integer.